Software/real_time/ROS_RoboBuggy/src/robobuggy/src/transistor/low_level/LL_Broadcaster.cpp

```diff
@@ -79,34 +70,65 @@ void Transistor_LL_Broadcaster::parse_serial_msg(std::string serial_msg) {
 
     // Add timestamps to messages
     ros::Time timestamp = ros::Time::now();
-    brake_msg.header.stamp = timestamp;
-    steering_msg.header.stamp = timestamp;
+    feedback_msg.header.stamp = timestamp;
     diagnostics_msg.header.stamp = timestamp;
     encoder_msg.header.stamp = timestamp;
 
     // Publish messages
-    publish_brake_msg(brake_msg);
-    publish_steering_msg(steering_msg);
+    publish_feedback_msg(feedback_msg);
     publish_diagnostics_msg(diagnostics_msg);
     publish_encoder_msg(encoder_msg);
 
 }
 
-int Transistor_LL_Broadcaster::handle_serial_messages() {
+void LL_Broadcaster::send_command(const robobuggy::Command::ConstPtr& msg) {
```

Comment:
I don't think this will be *too* much of a problem, but low level expects us to be sending these messages at a constant rate, and will time out if we don't send messages in time. In short, I think it would be a better idea for the command subscriber to update a class variable, and to send the commands periodically instead.

Reply:
Would it be a problem to move this into a separate feature branch and merge this PR without it? Might take me a bit longer to implement because I'm super busy this week and I'd like to get the cleanup stuff merged in to master soon.